packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1867,51 +1867,51 @@ export default class Meeting extends StatelessWebexPlugin {
     );
     Metrics.postEvent({event: eventType.MEDIA_RECONNECTING, meeting: this});
 
-    return this.reconnectionManager
-      .reconnect(this)
-      .then((reconnect) => {
-        Trigger.trigger(
-          this,
-          {
-            file: 'meeting/index',
-            function: 'reconnect'
-          },
-          EVENT_TRIGGERS.MEETING_RECONNECTION_SUCCESS,
-          {
-            reconnect
-          }
-        );
-        Metrics.postEvent({
-          event: eventType.MEDIA_RECOVERED,
-          meeting: this
-        });
+    let reconnect;
 
-        return Promise.resolve(reconnect);
-      })
-      .catch((error) => {
-        Trigger.trigger(
-          this,
-          {
-            file: 'meeting/index',
-            function: 'reconnect'
-          },
-          EVENT_TRIGGERS.MEETING_RECONNECTION_FAILURE,
-          {
-            error: new ReconnectionError('Reconnection failure event', error)
-          }
-        );
-        Metrics.postEvent({
-          event: eventType.MEDIA_RECONNECTION_FAILED,
-          meeting: this
-        });
+    try {
+      reconnect = await this.reconnectionManager.reconnect(this);
+      Trigger.trigger(
+        this,
+        {
+          file: 'meeting/index',
+          function: 'reconnect'
+        },
+        EVENT_TRIGGERS.MEETING_RECONNECTION_SUCCESS,
+        {
+          reconnect
+        }
+      );
+      Metrics.postEvent({
+        event: eventType.MEDIA_RECOVERED,
+        meeting: this
+      });
+    }
+    catch (error) {
+      Trigger.trigger(
+        this,
+        {
+          file: 'meeting/index',
+          function: 'reconnect'
+        },
+        EVENT_TRIGGERS.MEETING_RECONNECTION_FAILURE,
+        {
+          error: new ReconnectionError('Reconnection failure event', error)
+        }
+      );
+      Metrics.postEvent({
+        event: eventType.MEDIA_RECONNECTION_FAILED,
+        meeting: this
+      });
 
-        return Promise.reject(new ReconnectionError('Reconnection failure event', error));
-      })
-      .finally((reconnect) => {
-        this.reconnectionManager.reset();
+      this.reconnectionManager.reset();
 
-        return Promise.resolve(reconnect);
-      });
+      throw new ReconnectionError('Reconnection failure event', error);
+    }
+
+    this.reconnectionManager.reset();
```

Comment:
can we not do a try/catch/finally with the await? that is supported across all browsers